packs/data/bestiary-effects.db/effect-adaptive-strike.json

```diff
@@ -36,7 +36,8 @@
             {
                 "key": "Note",
                 "selector": "attack",
-                "text": "<p class='compact-text' data-visibility='gm'>{item|name}</p>"
+                "text": "{item|name}",
+                "visibility": "gm"
             }
```

Comment:
What's the deal with this note?

Reply:
![image](https://user-images.githubusercontent.com/64166996/175787057-635be109-90e9-49c0-87aa-a86ebbccf94e.png)

It appears that the thought was since the picked choice is added to the item name, just that is enough to determine what the note does. I'll tweak in a separate MR.